src/Microsoft.AspNet.Server.Kestrel/Infrastructure/KestrelThread.cs

```diff
@@ -64,15 +64,15 @@ public void Stop(TimeSpan timeout)
                 return;
             }
 
-            Post(OnStop, null);
```

Comment:
I would prefer making this PR a little small by changing this to:

    Post(t => t.OnStop());

Then the OnStop* methods don't need to be changed.